godot_project/python/scripts/openmm_server.py

```
@@ -861,7 +861,7 @@ def __init__(self, simulation_id, publish_scoket, publish_socket_lock, reportInt
 
 	def describeNextReport(self, simulation):
 		if self._has_error:
-			return 0
+			return (0, False, False, False, False, None)
```

this `return 0` was causing a silent error that was never informed back to MSEP